paasta_tools/tron_tools.py

```diff
@@ -304,7 +304,6 @@ def _build_spark_config(self) -> Dict[str, str]:
             "spark.kubernetes.executor.label.paasta.yelp.com/instance": truncated_instance,
             "spark.kubernetes.executor.label.paasta.yelp.com/cluster": self.get_cluster(),
             "spark.kubernetes.node.selector.yelp.com/pool": self.get_pool(),
-            "spark.kubernetes.executor.label.yelp.com/pool": self.get_pool(),
```

Comment:
don't you need to add the new label here?

Reply:
That would get added as a pod label for any spark executors launched by tron if that project had ever been finished @Rob-Johnson so it does not need to updated or added. The code is doing nothing.